web-src/src/components/TabsSearch.vue

```diff
@@ -56,15 +56,17 @@ export default {
 
   methods: {
     search_library: function () {
+      this.$store.state.search_path = '/search/library'
```

Comment:
Changing the state in a vuex store should be done with a "mutation" function: https://vuex.vuejs.org/guide/mutations.html